Lib/test/test_dataclasses.py

```diff
@@ -254,8 +254,10 @@ def test_field_named_object_frozen(self):
         @dataclass(frozen=True)
         class C:
             object: str
-        c = C('foo')
+            BUILTINS: int  # gh-96151
+        c = C('foo', 5)
         self.assertEqual(c.object, 'foo')
+        self.assertEqual(c.BUILTINS, 5)
```

Comment:
Please create a separate function for this, and don't mix it in with existing tests. Maybe `test_field_named_BUILTINS` or similar.

Reply:
Thanks for the review, made the change :-)